core/module.go

```diff
@@ -534,21 +765,21 @@ type SDK interface {
 
 	The provided Module is not fully initialized; the Runtime field will not be set yet.
 	*/
-	Codegen(context.Context, *Module, dagql.Instance[*Directory], string) (*GeneratedCode, error)
+	Codegen(context.Context, *Module, dagql.Instance[*ModuleSource]) (*GeneratedCode, error)
```

Comment:
Always nice when two args turn into one :man_cook::sunglasses::fire::pinched_fingers::kiss: (someday that'll merge into the standard guy fieri chef kiss emoji)